pkg/runner/fnscheduler/fnscheduler.go

```diff
@@ -58,10 +59,31 @@ type set struct {
 func (s *set) Invoke(parameters map[string]interface{}) (map[string]interface{}, error) {
 	if s.stats() > 0 {
 		// warm start, try to find a lowest latency process to work
-		s.Lock()
-		process := ChooseTargetInstance(s.instances)
-		s.Unlock()
-		return process.Invoke(parameters)
+		var result map[string]interface{}
+		var err error
+		err = retry.Do(
+			func() error {
+				s.Lock()
+				process := ChooseTargetInstance(s.instances)
+				s.Unlock()
+				result, err = process.Invoke(parameters)
+				zap.S().Debugw("retry in invoke err", "err", err)
+				return err
+			},
+			retry.RetryIf(func(err error) bool {
+				// this retry to avoid this case:
+				// one request choose the process when it is Running
+				// after the choose, the process Released
+				// the process will return instance.InstanceNotServiceErr to describe this case.
+				// todo thinking about the request is unlucky to retry at the fourth time.
```

Comment:
I think this scenario is a bit like OCC, so maybe the retry mechanism is enough

Wut do you think